tensorflow/lite/build_def.bzl

```diff
@@ -364,6 +364,7 @@ def _gen_selected_ops_impl(ctx):
         executable = ctx.executable._generate_op_registrations,
         mnemonic = "OpRegistration",
         progress_message = "gen_selected_ops",
+        use_default_shell_env = True,
```

Comment:
Thanks.

Wondering if this is actually causing an issue for you? 

I couldn't find the connection between this rule in tflite and the case you mentioned in the commit description.

Reply:
The case I mentioned is fixed by the change to `tensorflow/tensorflow.bzl`. However as with prior incarnations of this issue I patched all calls to `ctx.actions.run` and would expect also expect this issue to pop up in this rule by inspection:

It runs a previously compiled (IIUC) executable and hence depends on at least the compilers std-lib (e.g. `libstdc++.so`) but due to omitting `LD_LIBRARY_PATH` it will fail, which this change fixes.